src/main/java/com/minecolonies/coremod/client/gui/ViewFilterableList.java

```diff
@@ -50,7 +50,7 @@
     /**
      * String describing off for the gui.
      */
-    private static final String OFF = LanguageHandler.format(COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_RETRIEVE_OFF);
+    public static final String OFF = LanguageHandler.format(COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_RETRIEVE_OFF);
```

Comment:
Again why is this made public?
Expose your logic via method or change the behaviour.

Reply:
Also we should Refrain from using static fields which contain translated Content!
The user can change the language during runtime.